dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp

```diff
@@ -243,7 +243,8 @@ DeltaMergeStore::DeltaMergeStore(Context & db_context,
     try
     {
         page_storage_run_mode = storage_pool->restore(); // restore from disk
-        if (!storage_pool->maxMetaPageId())
+        if (const auto entry1 = storage_pool->metaReader()->getPageEntry(1);
```

Comment:
Let's give `1` a name.

Reply:
It's already has a name `DELTA_MERGE_FIRST_SEGMENT_ID`.